charts/seed-monitoring/charts/grafana/dashboards/operators/kubernetes-pods-dashboard.json

```diff
@@ -449,10 +453,38 @@
         "align": false,
         "alignLevel": null
       }
+    },
+    {
+      "datasource": "loki",
```

Comment:
The problem here is that the logging part is plugged into the existing monitoring dashboard (Logs and metrics in one place) and the monitoring can scrape metrics from the shoot. Our logging stack does not do this.

So we can not remove these these components from the selection field, because we will break working monitoring dashboards.

We can create another selection field for the logging but I am not sure if it is okay.